test/rpcgen_wrapper.sh

```diff
@@ -25,19 +25,20 @@ exit_failed() {
     echo "Could not generate regress.gen.\[ch\] using event_rpcgen.sh" >&2
     exit 1
 }
-
-if [ -x /usr/bin/python2 ] ; then
-  PYTHON2=/usr/bin/python2
-elif [ "x`which python2`" != x ] ; then
-  PYTHON2=python2
+if [ -x /usr/bin/python3 ] ; then
+  PYTHON=/usr/bin/python3
+elif [ -x /usr/bin/python2 ] ; then
+  PYTHON=/usr/bin/python2
+elif [ -x /usr/bin/python ] ; then
+  PYTHON=/usr/bin/python
 else
-  PYTHON2=python
+  PYTHON=python
 fi
 
 srcdir=$1
 srcdir=${srcdir:-.}
 
-${PYTHON2} ${srcdir}/../event_rpcgen.py --quiet ${srcdir}/regress.rpc \
+${PYTHON} ${srcdir}/../event_rpcgen.py --quiet ${srcdir}/regress.rpc \
                test/regress.gen.h test/regress.gen.c
```

Comment:
Maybe just replace all this `if/elif` with just allowing use shebang? It will became cleaner in this case.

Reply:
Also this PR misses changes into `test/test.sh` that invokes `check-dumpevents.py` and to `CMakeLists.txt` which also invokes it, can you take a look?